clients/admin-ui/src/features/poc/privacy-notices-sandbox/components/FetchPreferencesSection.tsx

```diff
@@ -0,0 +1,109 @@
+import {
+  AntButton as Button,
+  AntFlex as Flex,
+  AntInput as Input,
+  AntSelect as Select,
+  AntTypography as Typography,
+} from "fidesui";
+import palette from "fidesui/src/palette/palette.module.scss";
+
+import type { ConsentPreferenceResponse } from "~/types/api/models/ConsentPreferenceResponse";
+
+import PreviewCard from "./PreviewCard";
+
+interface FetchPreferencesSectionProps {
+  email: string;
+  onEmailChange: (email: string) => void;
+  selectedNoticeKeys: string[];
+  onSelectedNoticeKeysChange: (keys: string[]) => void;
+  parentNoticeKeys: Array<{ label: string; value: string }>;
+  onFetchCurrentPreferences: () => void;
+  isLoading: boolean;
+  isError: boolean;
+  error: any;
```

Comment:
One suggestion to avoid the use of `any` here is to set this as a `string` and do the logic happening on line 81 below on the `PrivacyNoticeSandboxRealData.tsx` file and pass in the message. That's where you have access to the error Type already, inferred from the RTK hook.